recipes/babelizer/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "babelizer" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/csdms/babelizer/archive/v{{ version }}.tar.gz
+  sha256: d1171b4ce1849911924389018b6135b343e2923122abb550565a91da0483c3a2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - babelize=babelizer.cli:babelize
+
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - black
+    - click
+    - gitpython
+    - pyyaml
+    - tomlkit
+    - isort >=5
+    - jinja2
+    - cookiecutter
```

Comment:
Are all of these needed to run pip?

Reply:
These are all dependencies of the package, but I suppose not needed to just run pip. Does this mean they should only be in the *run* section?

Reply:
@scopatz I've removed these requirements from the *host* section, but left them in the *run* section. I hope that's what you were getting at.

Thank you for looking at this so quickly! It's very much appreciated.